src/actions/passwordReset.action.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import CONSTANTS from '../constants';
+
+const { ERROR_ACTION, RESET_PASSWORD_ACTION } = CONSTANTS;
+
+const resetPasswordRequest = ({ email }) => dispatch => axios.post(`${process.env.REACT_APP_BASE_URL}/users/email_sent`, { email })
```

Comment:
Hadn't noticed this earlier
Use the axios instance: `api` that was created in a previous PR to make your api calls